### ISSUE TITLE
Drop dependency on fractal-demos

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-containers#92]
Because of using the fractal-demos script, we hit several complexities in tests here. The most relevant one is that fractal-demos scripts are meant to be portable and then use paths relative to their current directory, but it's not guaranteed that the client and server have access to shared filesystem (due to being in different containers, or more likely due to wrong permissions and different machine users).

Solving this while keeping the scripts identical to the fractal-demos ones introduces a lot of unnecessary complexity. We should rather copy the scripts here, and adapt them as needed - so that e.g. they write zarrs in `/home/test01/somewhere`.

**Comment by tcompa** [comment]
Note: the current solution is to use a specific fractal-demos branch (`reduce-memory-requirements-examples-01-02`).

**Comment by tcompa** [comment]
This becomes relevant again, as we would have to update fractal-demos due to breaking CLI-client changes in v2.18.0. But it's cumbersome to do so, since the CLI commands would be split (partly in this repo, partly in fractal-demos).

To do: extract a workflow JSON for examples 01 and 02 and store it in this repo, so that we fully drop the dependency on fractal-demos.

**Comment by jluethi** [comment]
Agreed with switching to using workflow jsons for our container tests!
